### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

**Comment by kke** [comment]
Fixored

File: lib/mortar/command.rb [hunk 1]
```diff
@@ -102,23 +104,21 @@ def variables_struct
...
         pkey.to_s.split(".").reverse.inject(pvalue) do |value, key|
-          {key.to_sym => value}
+          { key.to_sym => value }
         end
       end.inject(&:deep_merge)
     end
 
     # @return [K8s::Client]
     def client
-      return @client if @client
```
Comment (jakolehm) [inline lib/mortar/command.rb]: @kke why did we remove this?

Comment (kke) [reply lib/mortar/command.rb]: It was prettier than the type of syntax rubocop would have wanted:

```
@client = if ENV['xyz']
                    foofoo
                 elsif bar?
                    xyz
                 end
```

and the memoization was not used for anything.

**Comment by kke** [comment]
Conflicts resolved

**Pull request merged**